crates/apollo-compiler/test_data/diagnostics/0113_partially_overlapping_fragments.txt

```diff
@@ -0,0 +1,12 @@
+Error: operation must not select different types using the same field name `overlapping`
+    ╭─[0113_partially_overlapping_fragments.graphql:27:16]
+    │
+ 24 │     ... on B { overlapping }
+    │                ─────┬─────  
+    │                     ╰─────── `overlapping` has type `Int!` here
+    │ 
+ 27 │     ... on C { overlapping }
+    │                ─────┬─────  
+    │                     ╰─────── but the same field name has type `Int` here
```

Comment:
maybe this would be clarified by using the selection path, and saying that it is selected from both `B.overlapping: Int!` and `C.overlapping: Int`, without explicitly talking about "fields" and "response" anythings.